src/components/Footer/Footer.tsx

```diff
@@ -1,8 +1,33 @@
 import * as React from "react"
 import "./Footer.scss"
+import { FooterProps } from "Props/footer/props"
+import { ReactComponent as FooterHill } from "../../images/footerhill.svg"
 
```

Comment:
If the `footerProps` are constant between views, it might be better to import it within the component just so the view page is cleaner.

Reply:
Good point